crates/configuration/src/shared/helpers.rs

```diff
@@ -28,13 +28,14 @@ pub fn merge_errors_vecs(
 }
 
 pub fn ensure_node_name_unique(
-    node_name: String,
+    node_name: impl Into<String>,
```

Comment:
`AsRef<str>` might be an even better candidate across the crate for this, accepting both `&str` and `String`, but this is a nitpick

Reply:
Ahh yes, I will change in a follow pr. Thanks!